ext/POSIX/t/time.t

```diff
@@ -205,3 +205,8 @@ SKIP: {
     is(mktime(CORE::localtime($time)), $time, "mktime()");
     is(mktime(POSIX::localtime($time)), $time, "mktime()");
 }
+ 
+# Somewhat arbitrarily, put in 60 seconds of slack;  if this fails, it will
+# likely be off by 1 hour
+ok(abs(POSIX::strftime('%s', localtime) - time < 60),
```

Comment:
Misplaced parentheses. You want `abs(delta) < 60`, not `abs(delta < 60)`.

Apart from that, `strftime "%s"` returns seconds since 1970-01-01, but `perldoc -f time` says the time epoch is platform specific. In particular, I would expect this test to fail on Mac OS Classic (do we still support that?). No idea about VMS, DOS, or other "exotic" systems. Might need a `$^O` guard?

Reply:
Thanks

Reply:
experiment on Win32 yields the same.  Do we support DOS?  We can add skips as necessary, But I think this won't overwhelm us with cpan fails

Reply:
I don't think we support Mac classic

Reply:
Have to skip Windows and VMS, as strftime() doesn't handle `%s`